### DISCUSSION TITLE
Notable projects using taskfile?

### DISCUSSION THREAD

**Discussion by ssbarnea** [go-task/task#751] [Show and tell]
I tried to find a list of notable (big/popular) projects that use taskfile themselves but I was not able to find such a list.

The only finding so far was using https://sourcegraph.com/search?q=context:global+file:%5ETaskfile.ya%3Fml&patternType=regexp which returns results based on github stars. Only 3 projects seem to be more popular than taskfile itself:

* [gogs](https://github.com/gogs/gogs) 40k
* [drone](https://github.com/harness/drone) 25k
* [goreleaser](https://github.com/goreleaser/goreleaser) 10k

Maybe someone has better examples?

PS. Please note that used a special regex search for that as I did not want to include projects that might have a config in a subfolder that could happen to be just a vendored dependency.

**Comment by andreynering** [comment]
Hi @ssbarnea,

I think this project is probably not as big as you might think. (Or maybe another way to see it is that it still has a lot of space to grow).

To be honest, I'm impressed by the number of users it currently has, because in the beginning this was supposed to be a very small/simple tool, but evolved with time. I never imagined it would become this popular. Seems the interest has grown with time, if we take into account the [star history](https://star-history.com/#go-task/task).

There are other similar tools out there that are competent too, but didn't manage to get the same traction, like [Tusk](https://rliebz.github.io/tusk/), [Hammerkit](https://no0dles.gitbook.io/hammerkit/) and others, for comparison.

---

Regarding your question, I see many small repositories using Task, but not many big ones as you said. [Gitea](https://github.com/go-gitea/gitea) had conversations about adopting it in the past, but that didn't happen (at least yet). Between the relevant users that I know I'd like to highlight [Arduino](https://github.com/arduino/arduino-cli) and Microsoft with [this small repository](https://github.com/Azure/azure-service-operator). We should keep in mind that many may use Task on private repositories and we can't easily know about them.

**Comment by standardloop** [comment]
https://github.com/vercel/terraform-provider-vercel/blob/main/Taskfile.yml

Vercel's terraform provider repo is using taskfile

**Reply by standardloop** [reply]
Update, they moved to mise tasks:
- https://github.com/vercel/terraform-provider-vercel/blob/main/mise.toml
